.github/workflows/simulation.yml

```diff
@@ -62,6 +62,34 @@ jobs:
         ./bootstrap
         make -f examples/Makefile-simulation distcheck
 
+  cli-ftd-otns:
+    runs-on: ubuntu-18.04
+    env:
+      CFLAGS: -m32
+      CXXFLAGS: -m32
+      LDFLAGS: -m32
+      COVERAGE: 1
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 1
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.13'
+    - name: Bootstrap
+      run: |
+        sudo apt update
+        sudo apt --no-install-recommends install -y g++-multilib
+        ./bootstrap
+    - name: Run OTNS Tests
+      run: |
+        git clone https://github.com/openthread/ot-ns.git --depth 1 --branch master ~/otns
+        export OTDIR=$PWD
+        cd ~/otns
+        ./script/test py-unittests py-examples
+    - name: Codecov
```

Comment:
Should we also increase the value of `after_n_builds`? https://github.com/openthread/openthread/blob/fa96bcd10fef0e4ff9340bb331a5c7b10b96addf/.codecov.yml#L12